aten/src/ATen/native/native_functions.yaml

```
@@ -10221,6 +10221,17 @@
   python_module: linalg
   variants: function
 
+- func: linalg_cross(Tensor self, Tensor other, int dim=-1) -> Tensor
```

Array API specifies its `axis` argument as a keyword only. Let's have the same in PyTorch.
```suggestion
- func: linalg_cross(Tensor self, Tensor other, *, int dim=-1) -> Tensor
```

---

Done!